File: src/OrderPaper/wwwroot/app/models/OrderType.ts
import { Section, GroupItem } from './Section';

export class OrderType {
    IsFrontPage: boolean;
    Include: boolean;
    Name: string;
    Sections: Array<Section>;
    Groups: Array<Array<Section>>;
    GroupItems: Array<GroupItem>;
    constructor() {
        this.Sections = new Array<Section>();
        this.Groups = new Array<Array<Section>>();
        this.GroupItems = new Array<GroupItem>();
    }
}

File: src/OrderPaper/Models/OrderType.cs
using System.Collections.Generic;

namespace OrderPaper.Models
{
    public class OrderType
    {
        public bool IsFrontPage;
        public bool Include;
        public string Name;
        public List<Section> Sections;
        public List<List<Section>> Groups;
        public OrderType()
        {
            Sections = new List<Section>();
            Groups = new List<List<Section>>();
        }
    }
}
